fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/cursors/IteratorCursor.java

```diff
@@ -37,14 +39,40 @@
  */
 @API(API.Status.MAINTAINED)
 public class IteratorCursor<T> extends IteratorCursorBase<T, Iterator<T>> {
+    private static final Logger LOG = LoggerFactory.getLogger(IteratorCursor.class);
```

Comment:
```suggestion
    private static final Logger LOGGER = LoggerFactory.getLogger(IteratorCursor.class);
```

To match other usages in this project